### DISCUSSION TITLE
【夜间模式跟随系统】系统颜色与夜间模式出现 “反向”、“唱反调” BUG

### DISCUSSION THREAD

**Discussion by AWhiteC** [the1812/Bilibili-Evolved#1679] [综合讨论 / General]
![bug2](https://user-images.githubusercontent.com/40788545/112590072-2e8bdf80-8e3d-11eb-8f91-34b2da892e34.png)
![bug1](https://user-images.githubusercontent.com/40788545/112590085-3481c080-8e3d-11eb-960f-2c9dfe74c4b0.png)

**Comment by the1812** [comment]
你是不是 Edge? Edge 自身又弄了个亮暗设置覆盖系统的

**Reply by AWhiteC** [reply]
是的……刚刚看了一下浏览器设置页面，可能是后来 Edge Dev 升级导致的混乱……重新再设置就行了。问题已经解决，谢谢解答。
